src/component.coffee

```diff
@@ -227,7 +226,7 @@ Ember.AddeparMixins.ResizeHandlerMixin,
   # they can be. Note that this may fail to arrive at the table width if the
   # resizable columns are all restricted by min/max widths.
   doForceFillColumns: ->
-    allColumns = @get('allColumns')
+    allColumns = @get('columns')
```

Comment:
Looks like we don't use `allColumns` anywhere anymore; delete line 157 of this file?